packages/amplify-cli-core/src/errors/amplify-error.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Base class for all Amplify errors
+ */
+export class AmplifyError extends Error {
```

Comment:
I'm thinking it might be nice to have an `AmplifyException` base class with `AmplifyError` and `AmplifyFault` that extend it. It seems a little cumbersome to have to specify the `classification` every time we throw an error. Also I don't think we need a separate classification for ServiceCallFailure. It's still a Fault as it's something that wen't wrong that's out of the customer's control.